modules/ossm-cr-example.adoc

```diff
@@ -18,63 +18,257 @@ This example `ServiceMeshControlPlane` definition contains all of the supported
+        affinity: {}
+      container:
+        resources:
+          limits: {}
+          requirements: {}
 
-    kiali:
-      enabled: true
+  addons:
+    visualization:
+      grafana:
+        address: some-grafana-url 
+        install: 
+          selfManaged: true 
+          config:
+            env: {} 
+            envSecrets: {} 
+          persistence:
+            storageClassName: ""
+            accessMode: ReadWriteOnce
+            capacity: 5Gi 
+          service: 
+            ingress:
+              contextPath: /grafana
+              tls:
+                termination: reencrypt
+          runtime:
+            deployment: {}
+            pod: {}
+      kiali:
+        name: kiali 
+        install: # install kiali CR if not present
+          config:
+            dashboard:
+              viewOnly: false
+              enableGrafana: true
+              enableTracing: true
+              enablePrometheus: true
+          service: 
+            ingress:
+              contextPath: /kiali
+          runtime:
+            deployment: {}
+            pod: {}
+----
 
-    grafana:
-      enabled: true
+The following table lists the parameters for the `ServiceMeshControlPlane` resource.
 
-    tracing:
-      enabled: true
-      jaeger:
-        template: all-in-one
-----
+.`ServiceMeshControlPlane` resource parameters
+|===
+|Name |Description |Type
+
+|`apiVersion`
+|APIVersion defines the versioned schema of this representation of an object. Servers convert recognized schemas to the latest internal value, and may reject unrecognized values. The value for {ProductName} version 2.0 is `maistra.io/v2`.
+|The value for {ProductName} version 2.0 is maistra.io/v2.
+
+|kind
+|Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated.
+|CamelCase string
+
+|metadata
+|Metadata about this `ServiceMeshControlPlane` instance You can provide a name for your control plane installation to keep track of your work, for example, `basic-install`, or `production`.
+|string
+
+|spec
+|The specification of the desired state of this `ServiceMeshControlPlane`. This includes the configuration options for all components that comprise the control plane.
+|For more information, see Table 2.
+
+|status
+|The current status of this `ServiceMeshControlPlane` and the components that comprise the control plane. 
+|For more information, see `ControlPlaneStatus`.
+|===
+
+The following table lists the specifications for the `ServiceMeshControlPlane` resource. These parameters configure {ProductName} for your microservices and apps.
+
+.`ServiceMeshControlPlane` resource spec
+|===
+|Name |Description |Configurable parameters
+
+|addons
+|Addons is used to configure additional features beyond core control plane components, such as visualization, or metric storage.
+|`3scale`, `grafana`, `jaeger`, `kiali`, `prometheus`, and `stackdriver`
```

Comment:
Do we support stackdriver?  Or is that just upstream, since it's a Google offering?

Reply:
we do not